An important change

AMZN-alexpete opened the pull request.

amzn-changml committed:
Create codeql-analysis.yml

Signed-off-by: Mike Chang <changml@amazon.com>

amzn-changml committed:
Initial commit of DCO checker

Signed-off-by: Mike Chang <changml@amazon.com>

amzn-changml committed:
Merge branch 'o3de:development' into development

amzn-changml committed:
Update job label

Signed-off-by: Mike Chang <changml@amazon.com>

AMZN-alexpete committed:
An important change

amzn-changml commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # O3DE (Open 3D Engine)
 
-O3DE (Open 3D Engine) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
+O3DE (Open 3D Elephant) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
```
```suggestion
O3DE (Open 3D Giraffe) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
```

AMZN-alexpete replied:
"Sign off and commit suggestion" nice!  
![image](https://user-images.githubusercontent.com/26804013/171258595-60988258-d5d5-44fd-b619-d7053222c4be.png)

amzn-changml committed:
Update pytest.ini

Signed-off-by: Mike Chang <changml@amazon.com>

amzn-changml committed:
Update pytest.ini

Third-Party DCO Remediation Commit for Alex Peterson <26804013+AMZN-alexpete@users.noreply.github.com>

On behalf of Alex Peterson <26804013+AMZN-alexpete@users.noreply.github.com>, I, Mike Chang <changml@amazon.com>, hereby add my Signed-off-by to this commit: ab37af1ca643cc3de1f352aa32d569844132b39c

Signed-off-by: Mike Chang <changml@amazon.com>

Signed-off-by: Mike Chang <changml@amazon.com>
